packages/pipboard-client/ice.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   proxy: {
     '/**': {
       enable: true,
-      target: 'http://127.0.0.1:7001',
+      target: 'http://127.0.0.1:6927',
```

Comment:
IMO it's good to import from pipcook-core :)

Reply:
proxy config of ice is just for local dev. It will not be used in production env. I think we do not need pipboard to depend on core just for this single item.